packages/phoenix-otel/src/phoenix/otel/otel.py

```diff
@@ -22,17 +23,19 @@
 
 PROJECT_NAME = _ResourceAttributes.PROJECT_NAME
 
+_DEFAULT_GRPC_PORT = 4317
+
 
 def register(
     *,
     endpoint: Optional[str] = None,
     project_name: Optional[str] = None,
     batch: bool = False,
-    set_global=True,
-    headers=None,
+    set_global_tracer: bool = True,
```

Comment:
```suggestion
    set_global_tracer_provider: bool = True,
```

I guess more explicit maybe